packages/ui/src/theme/defaultDarkMode.ts

```diff
@@ -0,0 +1,115 @@
+        90: { value: tokens.colors.orange[20].value },
+        100: { value: tokens.colors.orange[10].value },
+      },
+      yellow: {
+        10: { value: tokens.colors.yellow[100].value },
+        20: { value: tokens.colors.yellow[90].value },
+        40: { value: tokens.colors.yellow[80].value },
+        // 60 doesn't change
+        80: { value: tokens.colors.yellow[40].value },
+        90: { value: tokens.colors.yellow[20].value },
+        100: { value: tokens.colors.yellow[10].value },
+      },
+      green: {
+        10: { value: tokens.colors.green[100].value },
+        20: { value: tokens.colors.green[90].value },
+        40: { value: tokens.colors.green[80].value },
+        // 60 doesn't change
+        80: { value: tokens.colors.green[40].value },
+        90: { value: tokens.colors.green[20].value },
+        100: { value: tokens.colors.green[10].value },
+      },
+      teal: {
+        10: { value: tokens.colors.teal[100].value },
+        20: { value: tokens.colors.teal[90].value },
+        40: { value: tokens.colors.teal[80].value },
+        // 60 doesn't change
+        80: { value: tokens.colors.teal[40].value },
+        90: { value: tokens.colors.teal[20].value },
+        100: { value: tokens.colors.teal[10].value },
+      },
+      blue: {
+        10: { value: tokens.colors.blue[100].value },
+        20: { value: tokens.colors.blue[90].value },
+        40: { value: tokens.colors.blue[80].value },
+        // 60 doesn't change
+        80: { value: tokens.colors.blue[40].value },
+        90: { value: tokens.colors.blue[20].value },
+        100: { value: tokens.colors.blue[10].value },
+      },
+      purple: {
+        10: { value: tokens.colors.purple[100].value },
+        20: { value: tokens.colors.purple[90].value },
+        40: { value: tokens.colors.purple[80].value },
+        // 60 doesn't change
+        80: { value: tokens.colors.purple[40].value },
+        90: { value: tokens.colors.purple[20].value },
+        100: { value: tokens.colors.purple[10].value },
+      },
+      pink: {
+        10: { value: tokens.colors.pink[100].value },
+        20: { value: tokens.colors.pink[90].value },
+        40: { value: tokens.colors.pink[80].value },
+        // 60 doesn't change
+        80: { value: tokens.colors.pink[40].value },
+        90: { value: tokens.colors.pink[20].value },
+        100: { value: tokens.colors.pink[10].value },
+      },
+      neutral: {
+        10: { value: tokens.colors.neutral[100].value },
+        20: { value: tokens.colors.neutral[90].value },
+        40: { value: tokens.colors.neutral[80].value },
+        // 60 doesn't change
+        80: { value: tokens.colors.neutral[40].value },
+        90: { value: tokens.colors.neutral[20].value },
+        100: { value: tokens.colors.neutral[10].value },
+      },
+      black: { value: '#fff' },
+      white: { value: '#000' },
+
+      border: {
+        primary: { value: '{colors.neutral.40.value}' },
```

Comment:
Nice! neutral-40 will meet the contrast ratio requirement in dark mode. I have a task that is related to some changes here. Would like to get this in ASAP so I can audit and work base off it.